exasol/nb_connector/text_ai_extension_wrapper.py

```diff
@@ -173,76 +199,119 @@ def initialize_text_ai_extension(
             Optional. Text-AI extension version.
         language_alias:
             The language alias of the extension's language container.
-        run_deploy_container:
+        install_slc:
             If True runs deployment of the locally stored Script Language Container file for the Text-AI Extension.
-        run_deploy_scripts:
+        install_udf_scripts:
             If True runs deployment of Text-AI Extension scripts.
-        run_upload_models:
+        install_models:
             If True uploads default Transformers models to the BucketFS.
-        run_encapsulate_bfs_credentials:
+        install_bfs_credentials:
             If set to False will skip the creation of the text ai specific database connection
             object encapsulating the BucketFS credentials.
-        allow_override:
+        allow_override_language_alias:
             If True allows overriding the language definition.
     """
 
-    # Create the name of the Exasol connection object
-    db_user = str(conf.get(CKey.db_user))
-    bfs_conn_name = "_".join([BFS_CONNECTION_PREFIX, db_user])
-    # As soon as the official release of TXAIE is available, the hard-coded value for
-    # container_name can be replaced by TXAIELanguageContainerDeployer.SLC_NAME,
-    # see https://github.com/exasol/notebook-connector/issues/179.
-    container_name = "exasol_text_ai_extension_container_release.tar.gz"
-
-    def from_ai_lab_config(key: CKey) -> Path | None:
-        entry = conf.get(key)
-        return Path(entry) if entry else None
-
-    if run_deploy_container:
-        if version:
-            install_text_ai_extension(version)
-            # Can run_upload_models, run_deploy_scripts,
-            # run_encapsulate_bfs_credentials, etc. be ignored here?
-            return
-
-        container_file = container_file or from_ai_lab_config(
-            CKey.txaie_slc_file_local_path
+    def deploy_slc(container_file=None, version=None):
+        container_url = version and TXAIELanguageContainerDeployer.slc_download_url(
+            version
         )
-        if not container_file:
-            install_text_ai_extension(LATEST_KNOWN_VERSION)
+        deploy_language_container(
+            conf,
+            container_name=TXAIELanguageContainerDeployer.SLC_NAME,
+            container_file=container_file,
+            container_url=container_url,
+            language_alias=language_alias,
```

Comment:
```suggestion
            language_alias=LANGUAGE_ALIAS,
```

we use in the Extraction the constant